lib/src/auth/auth_login_form.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'auth_controller.dart';
+
+class AuthLoginForm extends StatefulWidget {
+  const AuthLoginForm({Key? key}) : super(key: key);
+
+  @override
+  State<AuthLoginForm> createState() => _AuthLoginFormState();
+}
+
+class _AuthLoginFormState extends State<AuthLoginForm> {
+  final _formKey = GlobalKey<FormState>();
+  late String _email;
+  late String _password;
+
+  void _submit(BuildContext context) async {
```

Comment:
Since this is a Stateful Widget , you don't need to pass the context throw parameter, you already have access to it.

Reply:
Resolved